lib/twitter.js

```diff
@@ -161,7 +161,7 @@ Twitter.prototype.post = function(url, content, content_type, callback) {
     }
     else {
       try {
-        var json = JSON.parse(data || {});
+        var json = data ? JSON.parse(data) : data;
         callback(null, json);
```

Comment:
I think it's not a problem, because if no data is received here `JSON.parse({})` will throw an exception.